src/components/FlexGrid/License.md

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I've emailed legal to confirm this is the best way to handle this as they listed MIT license in package.json and README, but did not include the customary License file. So I just need to confirm my taking liberty to add it here myself is the correct move.